docs/api.rst

```diff
@@ -30,7 +30,8 @@ elasticsearch returns a 2XX response. Otherwise an instance of
 raised. You can see other exception and error states in :ref:`exceptions`. If
 you do not wish an exception to be raised you can always pass in an ``ignore``
 parameter with either a single status code that should be ignored or a list of
-them::
+them
+.. code-block:: python
```

Comment:
Typically there's a blank line between text and a directive.

Reply:
Done